components/Navbar/UserButton.tsx

```diff
@@ -19,14 +19,14 @@ export default function UserButton() {
 
     return isLoggedIn ? (
       <button
-        onClick={() => setOpenSignInModal(true)}
+        onClick={() => setOpenSignInModal(true) aria-label="profile"}
         className="text-grey-300 ml-4 h-10 w-10 cursor-pointer"
       >
         <Avatar avatar={user.avatar} size={30} />
       </button>
     ) : (
       <button
-        onClick={() => toggleModal(true)}
+        onClick={() => toggleModal(true) aria-label="profile"}
```

Comment:
same here the aria-label needs to be outside of the closing curly-bracket
`onClick={() => toggleModal(true)}
aria-label="profile"`

Reply:
Hey, thanks @benalleng. Will change it as you mentioned :)